src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3874,6 +3862,37 @@ EOF
   }
 }
 
+#======================================================================
+sub get_nens { 
+    #default 
+    $atm_nens = 32;
+    $ldas_nens_det = 24;
+    $ldas_nens_atmens = $atm_nens; 
+    
+    @enslist = (<$fvhome/atmens/mem???>);
+    if ( scalar(@enslist) > 1 ) {
+       $atm_nens = scalar(@enslist);
+    }
+    $ldasdomain ="CF$agcm_im4x6C_GLOBAL";
+    $ldasics_det = "$ldasics/output/$ldasdomain";
+    @enslist = (<$ldasics_det/rs/ens????>);
+    if ( scalar(@enslist) > 1 ) {
+        $ldas_nens_det = scalar(@enslist);
+    }
+    $aens_im4 = sprintf "%04i", $aens_im ; 
+    $ldasdomain ="CF$aens_im4x6C_GLOBAL";
+    $ldasics_ens = "$ldasics4ens/output/$ldasdomain";
+    @enslist = (<$ldasics4ens/rs/ens????>);
+    if ( scalar(@enslist) > 1 ) {
+       $ldas_nens_atmens = scalar(@enslist);
+    }
+    if ( $atm_nens ne  $ldas_nens_atmens ) {
```

Comment:
this check probably doesn't really do anything meaningful unless we run get_nens() after the atm ens has been set up (because $atm_nens is set to its default value)